Database/src/main/resources/changelog1.xml

```diff
@@ -781,4 +781,36 @@
+            <column name="internal_name" value="DocumentWorkerHandler"/>
+            <column name="conflict_resolution_mode" value="PRIORITY"/>
+            <column name="project_id"/>
+        </insert>
+    </changeSet>
+
+    <changeSet author="CAF" id="1512411132000-2">
+        <insert tableName="tbl_policy_type">
+            <column name="item_name" value="Composite Document Policy Type"/>
+            <column name="description" value="Enriches a family of documents."/>
+            <column name="definition" value="{&quot;title&quot;:&quot;Composite Document Policy Type&quot;,&quot;description&quot;:&quot;Allows the Policy Worker to integrate with workers which meet the Document Worker contract.  Document Workers can add extra fields to a document, and can also update or remove fields.&quot;,&quot;type&quot;:&quot;object&quot;,&quot;properties&quot;:{&quot;workerName&quot;:{&quot;description&quot;:&quot;The name of the worker. Some configuration details, such as the default queue names, will be looked up based on this name.&quot;,&quot;type&quot;:&quot;string&quot;},&quot;queueName&quot;:{&quot;description&quot;:&quot;Optional. Typically the target queue name is provided via configuration. This field can be used as a means to override the queue name specified in the configuration.&quot;,&quot;type&quot;:&quot;string&quot;},&quot;diagnosticsQueueName&quot;:{&quot;description&quot;:&quot;Optional. Allows a second 'diagnostic' queue to be specified for the worker. This will be used if the worker fails.&quot;,&quot;type&quot;:&quot;string&quot;}},&quot;customData&quot;:{&quot;description&quot;:&quot;Optional. Used to send additional worker-specific data to the worker.&quot;,&quot;type&quot;:&quot;object&quot;}}"/>
```

Comment:
Should `customData` not be inside `properties`?

Reply:
It should, looks like an error in the policy definition.